examples/amp-list.state.html

```diff
@@ -0,0 +1,68 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <script async custom-element="amp-list" src="https://cdn.ampproject.org/v0/amp-list-0.1.js"></script>
+  <script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-latest.js"></script>
+  <script async custom-element="amp-bind" src="https://cdn.ampproject.org/v0/amp-bind-0.1.js"></script>
+  <link rel="canonical" href="https://amp.dev/documentation/examples/components/amp-list/">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <style amp-custom>
+    amp-list {
+      margin-left: 16px;
+    }
+    .list-overflow {
+      width: 160px;
+      margin-left: auto;
+    }
+  </style>
+</head>
+<body>
+
+  <div on="tap:list1.refresh,list2.refresh,list3.refresh">
+    Refresh All Lists
+  </div>
+
+  <h3>Static amp-list, inlined state.
```

Comment:
You can consider adding this to `examples/bind/list.amp.html`. Also feel free to reorganize these example files if you'd like. They're pretty messy.

Reply:
It feels as though it equally belongs with `amp-list` and `amp-bind`, but potentially more so with `amp-list`. Mind if I move all of the amp-list examples to `examples/amp-list/*.html` after this PR goes through?